flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/DefaultContextUtils.java

```diff
@@ -51,12 +52,17 @@ public static DefaultContext buildDefaultContext(CliOptions.EmbeddedCliOptions o
         } else {
             libDirs = Collections.emptyList();
         }
-        return DefaultContext.load(
-                options.getPythonConfiguration(), discoverDependencies(jars, libDirs), true);
+        Configuration pythonConfiguration = options.getPythonConfiguration();
+        pythonConfiguration.addAll(
+                ConfigurationUtils.createConfiguration(options.getSessionConfig()));
+        return DefaultContext.load(pythonConfiguration, discoverDependencies(jars, libDirs), true);
```

Comment:
nit: the name is not very suitable here. How about sessionConfig here?

Reply:
LGTM, DONE